larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -219,7 +269,8 @@ class BaseBConv2DOpModel : public SingleOpModel {
         fbb.Int(1);
       });
       fbb.String("filter_format", "OHWI");
-      fbb.String("padding", padding == Padding_VALID ? "VALID" : "SAME");
+      fbb.String("padding", GetPaddingName(padding));
```

Comment:
`GetPaddingName` still supports `SAME_ONE` but `BConv` op does not support that type and instead we have the pad_values. so we might pass the wrong argument to `BConv` op by using `GetPaddingName` utility function right?

Reply:
Correct. I still left it in `GetPaddingName` because its used in the testing-string.
It might be cleaner to change `GetPaddingName` to only have `VALID` and `SAME` and use a seperate thing `pv0` & `pv1` (pad value 0/1) in the testing-string.
What do you think?

Reply:
yes, it should be removed from the padding type and add a separate substring for for tests.